lib/ansible/modules/blockinfile.py

```diff
@@ -274,9 +284,9 @@ def main():
         original = None
         lines = []
     else:
-        with open(path, 'rb') as f:
+        with open(path, 'r', encoding=encoding, newline='') as f:
             original = f.read()
-        lines = original.splitlines(True)
+            lines = original.splitlines(True)
```

Comment:
This indentation change is not required.


```suggestion
        lines = original.splitlines(True)
```